plugin/hosts/setup.go

```diff
@@ -25,6 +26,33 @@ func setup(c *caddy.Controller) error {
 		return plugin.Error("hosts", err)
 	}
 
+	parseChan := make(chan bool)
+
+	c.OnStartup(func() error {
+		h.ReadHosts()
+		if err != nil {
+			return err
+		}
+
+		go func() {
+			ticker := time.NewTicker(5 * time.Second)
```

Comment:
This deserves a small sentence in the README.md (e.g. "We scan for changes every 5s." or something)